.github/workflows/check_execute

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import subprocess, sys, argparse, time
+
+parser = argparse.ArgumentParser(description="Check the startup of the execute container")
```

Comment:
nbd but if we can configure the image name, maybe we don't want to specify what container is having its startup checked? Or maybe we don't need to make the image name configurable? No changes required here, just thinking out loud

Reply:
I can interpolate the image. Seems like the right call there.